src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/court/VideoLinkBookingService.kt

```diff
@@ -339,4 +343,27 @@ class VideoLinkBookingService(
       videoLinkBookingEventListener.appointmentUpdatedInNomis(videoLinkAppointment, appointmentChangedEventMessage)
     }
   }
+
+  fun deleteAppointments(releasedOffenderEventMessage: ReleasedOffenderEventMessage) {
+    val reason = releasedOffenderEventMessage.additionalInformation.reason
+    if (reason == Reason.TRANSFERRED || reason == Reason.RELEASED) {
+      val offenderBookings = prisonApiService.getOffenderDetailsFromOffenderNos(
+        listOf(releasedOffenderEventMessage.additionalInformation.nomsNumber),
+        false,
+      )
+      if (offenderBookings.isNotEmpty()) {
+        val prisonerAppointments =
+          videoLinkAppointmentRepository.findAllByHearingTypeIsAndStartDateTimeIsAfterAndVideoLinkBookingOffenderBookingIdIsAndVideoLinkBookingPrisonIdIs(
+            hearingType = MAIN,
+            startDateTime = LocalDateTime.parse(
+              releasedOffenderEventMessage.occurredAt,
+              DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss'Z'"),
+            ),
+            offenderBookingId = offenderBookings.first().bookingId,
```

Comment:
is this right? How do you know it will be the first booking returned?  Looking at https://github.com/ministryofjustice/prison-api/blob/17d18b986f66d7881fffe17ea2999860cfdd2f7b/src/main/java/uk/gov/justice/hmpps/prison/repository/sql/InmateRepositorySql.kt#L91 there is no sort order on the query

Reply:
the response from prisonApi should be a List of 1 item as we have a query string of activeOnly which we have set to true. 
/bookings/offenders?activeOnly={isActive}
This is based on the (reasonable?) assumption that there is only 1 active booking.